ssl/quic/quic_ackm.c

```diff
@@ -61,7 +61,8 @@ DEFINE_LHASH_OF_EX(OSSL_ACKM_TX_PKT);
 
 static unsigned long tx_pkt_info_hash(const OSSL_ACKM_TX_PKT *pkt)
 {
-    return pkt->pkt_num;
+    /* We assume that unsigned long is 64 bits */
+    return (unsigned long)pkt->pkt_num;
```

Comment:
This is fine but remove the comment. We don't assume unsigned long is 64 bits, but it's the API of LHASH, so it's what we need to use. We assume taking the low bits of pkt_num is an acceptable hash for our purposes, which it should always be given any remotely likely distribution of packet numbers. You could add a comment like "use low bits of the packet number as the hash" if you like.

Reply:
I'll fix the comment